tests/doc_test/add_sections/conf.py

```diff
@@ -41,15 +41,7 @@
     "impacts": directives.unchanged
 }
 
-cwd = os.getcwd()
-plantuml = 'java -jar %s' % os.path.join(cwd, "../utils/plantuml_beta.jar")
-
-# If we are running on windows, we need to manipulate the path,
-# otherwise plantuml will have problems.
-if os.name == "nt":
-    plantuml = plantuml.replace("/", "\\")
-    plantuml = plantuml.replace("\\", "\\\\")
-
+plantuml = 'java -jar %s' % os.path.join(os.path.dirname(__file__), "..", "utils", "plantuml_beta.jar")
```

Comment:
rather than define this in every file, do you think it might be better to define it *once* and then import it into each file?
that way if it changes, you only need to change it one place.

admittedly you *could* do it with a find + replace instead

Reply:
This can only bee achived when  messing around with ``sys.path`` to have have such a helper file importable everywhere you need it. I don't like this, but if @danwos likes this solution, we can do it that way. Personally I would not change it. The current implementation is simple, it works and it is unlikely that this will change.

Reply:
I'm not sure I understand. Why does it involve messing with sys.path?

What happens if you calculate the absolute path *once* and then pass it around as a constant?

Reply:
Surely you would just stick an __init__.py file in the tests/ root and then use a relative import?